tests/spec/language/translate.spec.tsx

```diff
@@ -22,7 +22,18 @@ describe('Translate', () => {
           foofoo: 'text {{ foo}} this {{bar}} <b>test</b>',
           foodirty: 'text {{ foo }} this {{bar}} <script>execute</script><br/><hr><div>test</div> <a href="test">link</a>',
           'baz.foo': 'dirty key',
-          foozfooz: 'jhipster is <strong>awesome</strong>!'
+          foozfooz: 'jhipster is <strong>awesome</strong>!',
+          'bar1.bar2': {
```

Comment:
Do we have such paths? I remember we only had 'foo.bar' like stuff, which were left over from legacy code

Reply:
I don't think we have this kind of path, but as this new code takes care of this, I think it's good.
Here a sample of what we currently have: https://github.com/jhipster/generator-jhipster/blob/master/generators/languages/templates/src/main/webapp/i18n/en/global.json.ejs#L148-L150

Reply:
Yes, that was my point, why add this code if we don't have this kind of path? and we should never add this kind of path. So why fix something that is not broken

Reply:
it's not a code here, it's a test (as it's in spec file), so now we're sure this kind of thing can't broke our front anymore

Reply:
I mean the test is fine but I don't see what is the bug here